generated/nifake/nifake/GrpcOptions.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from enum import Enum
+
+
+class SessionInitializationBehavior(Enum):
+    UNSPECIFIED = 0
+    r'''
+    It is acceptable to either initialize a new session or attach to an existing one.
+    '''
+    INITIALIZE_NEW = 1
+    r'''
+    This requires the NI gRPC Device Server to initialize a new session with the specified name.
+    '''
+    ATTACH_TO_EXISTING = 2
+    r'''
+    This requires the NI gRPC Device Server to attach to an existing session with the specified name.
+    '''
+
+
+class GrpcOptions(object):
```

Comment:
Consider naming this `GrpcDeviceOptions` to disambiguate it from gRPC channel/server options such as `grpc.insecure_channel(target, options=[("grpc.max_receive_message_length", -1)])`

Reply:
The server has the word "Device" in the name but I don't like including that in all the clients. It feels weird for DAQmx tasks or XNET sockets. Would `GrpcSessionOptions` work? Thoughts @DavidCurtiss ?

Reply:
Sure. `GrpcSessionOptions` also describes its purpose more clearly.